lib/scala/kg/src/main/scala/io/github/tetherlessworld/mcsapps/lib/kg/stores/postgres/AbstractPostgresKgStore.scala

```diff
@@ -3,10 +3,16 @@ package io.github.tetherlessworld.mcsapps.lib.kg.stores.postgres
 import play.api.db.slick.{DatabaseConfigProvider, HasDatabaseConfigProvider}
 import slick.jdbc.PostgresProfile
 
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, Future}
+
 
 abstract class AbstractPostgresKgStore(protected val dbConfigProvider: DatabaseConfigProvider) extends HasDatabaseConfigProvider[PostgresProfile] {
   import profile.api._
 
+  protected val SentencesDelimChar = '|'
+  protected val SentencesDelimString = SentencesDelimChar.toString
+
   lazy val kgEdges = TableQuery[KgEdgeTable]
```

Comment:
Remove the prefix from internal constructs like this. (Keep it on the outer class names like AbstractPostgresKgStore and PostgresKgQueryStore.)

Reply:
Gotcha, prefixes removed